packages/codemods/.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["plugin:@fluentui/eslint-plugin/node"],
+  "extends": ["plugin:@fluentui/eslint-plugin/node", "plugin:@fluentui/eslint-plugin/react"],
```

Comment:
looks like `--legacy` isn't being used as it should. this is v8 package this shouldn't use v9 lint rules at all. This would need additional refactoring.